credit.c

```diff
@@ -1,29 +1,83 @@
 #include <cs50.h>
 #include <stdio.h>
 
+// good job with formating. this is something that a lot of beginners mess up
+// and it shows that you're not a noob when you know how to format.
+// i personally like to put my open curly braces on the same line that i init my
+// function / if / while blocks but either works. if you work in an existing codebase,
+// follow the code formating rules they use.
+
+// ex: i do something like 
+// if (True == True) {
+//     printf("True");
+// }
+
+// in general try to separate your code in functions. it makes your code much more readible
+// and easier to debug
+
 int main(void)
 {
-    // Obtain user input for credit card number
-    long Number;
-    do
-    {
-        Number = get_long("Number: ");
-    }
-    while (Number < 0);
+    // i really like that you defined all your variables at the top
+    // it makes your code easier to read when you want to figure out what
+    // type a variable is. this is a good pattern in any language where you
+    // have to define variables before you use them
+
+    // your variables names should be consistent. pick either snake_case or camel_case
+    // and always use all capitial letters + underscores for constant variables
+    // Number => Number
+    // checksum1 => checkSum1
+    // checksum2 => checkSum2
+
+    // different languages have different preferences. Javascript prefers camel case.
+    // Python prefers snake case. In C, i dont think it matters just be consistent
 
+    // look up how to change all variables names in vscode. super useful tool.
+
+    // Obtain user input for credit card Number
+    long Number;
     // Find first checksum
     // (sum of digits of twice every other digit from second to last)
-    long Number2 = Number;
-    long checkNum = Number;
+    long Number2;
+    long checkNum;
     long checksum1 = 0;
     long checksum2 = 0;
     long subtractor;
     long adder;
     long counter = 10;
 
+    // you should make a const variable for this constant because you use it so often.
+    // it's not recommended you keep typing it out because there's a high change you'll
+    // accidentally make a typo and it makes it easier to change in one location if you need to
+    // long CREDIT_MODULUS = 1000000000000000
+
+    // most people don't use do while loops anymore. i would avoid using them. ill admit
+    // this is a great use for a do while loop. i wouldn't use them ever tho
+    // also move this block below your other definition statements for readibility
     do
     {
-        subtractor = Number % (counter);
```

Comment:
on github, sometimes the changes wouldn't reflect accurately. i modified 
```
subtractor = Number % (counter);
```
to 
```
subtractor = Number % counter;
```
on line 80